Altis_Life.Altis/dialog/function/fn_gangWithdraw.sqf

```diff
@@ -6,16 +6,15 @@
     Description:
     Withdraws money from the gang bank.
 */
-private ["_value"];
-_value = parseNumber(ctrlText 2702);
-_gFund = GANG_FUNDS;
+private _value = parseNumber(ctrlText 2702);
+private _gFund = GANG_FUNDS;
 group player setVariable ["gbank_in_use_by",player,true];
 
 //Series of stupid checks
 if (isNil {(group player) getVariable "gang_name"}) exitWith {hint localize "STR_ATM_NotInGang"}; // Checks if player isn't in a gang
 if (_value > 999999) exitWith {hint localize "STR_ATM_WithdrawMax";};
 if (_value < 0) exitWith {};
-if (!([str(_value)] call TON_fnc_isnumber)) exitWith {hint localize "STR_ATM_notnumeric"};
+if !([str(_value)] call TON_fnc_isnumber) exitWith {hint localize "STR_ATM_notnumeric"};
```

Comment:
This could simply be ``!(str _value call TON_fnc_isNumber)``, in the if of course ;)